pynxtools/dataconverter/template.py

```diff
@@ -175,8 +175,12 @@ def __delitem__(self, key):
         if key in self.optional.keys():
             del self.optional[key]
 
-        if key in self.required.keys():
```

Comment:
I would say it would not generate any side-effect,if nobody delete any required concept. In case, the required concept is deleted the validation of the converter validation step. As our converter validates each concept by backtracking according to the application definition.